cnf-certification-test/lifecycle/suite.go

```diff
@@ -316,6 +323,10 @@ func testDeploymentScaling(env *provider.TestEnvironment, timeout time.Duration)
 	defer env.SetNeedsRefresh()
 	failedDeployments := []string{}
 	for i := range env.Deployments {
+		if env.Deployments[i].OwnerReferences != nil {
```

Comment:
Thinking more about it, I feel it would be more reliable if we add a new entry in tnf_config.yaml to indicate the statefulsets and deployments that are scaled/managed by another process. The new section could be a list of sub-string that could help identify deployments and statefulsets that should not be scaled directly:
ManagedDeploymentsStatefulsets:
- test123
- abcs

The owner reference does not guarantee that the referenced owner will be able to scale this deployment. Also the owner reference does not necessarily imply that the deployment should not be scaled directly.
Maybe we can add a separate test to catch all the deployments or statefulsets that are not scalable either by CR or directly. 
In this test:
if the deployment/statefulset matches one in the list of "ManagedDeploymentsStatefulsets" skip this test, other wise run it

In the new optional test "test that deployments and statefulsets are scalable via a standard scaling API ":
- if a deployment is managed but the owner reference is not in the list of CRs that are scalable -> fail the test
- Otherwise pass it